ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/qos_profiles.schema.yml

```diff
@@ -144,3 +144,95 @@ keys:
                     description: |
                       Supported options are platform dependent
                       Example: "< rate > kbps", "1-100 percent", "< rate > pps"
+        priority_flow_control:
+          type: dict
+          description: |
+            Priority Flow Control settings
+          keys:
+            enabled:
+              type: bool
+              description: |
+                Enable Priority Flow control.
+            watchdog:
+              type: dict
+              description: |
+                Watchdog can detect stuck transmit queues.
+              keys:
+                enabled:
+                  type: bool
+                  required: true
+                  description: |
+                    Enable the watchdog on stuck transmit queues.
+                action:
+                  type: str
+                  valid_values: ["drop", "notify-only"]
+                  description: |
+                    Override the default ERRDISABLE action to either DROP
+                    traffic on the stuck queue or notify-only
+                    without making any actions on the stuck queue.
+                timer:
+                  type: dict
+                  description: |
+                    Timer thresholds whilst monitoring queues.
+                  keys:
+                    timeout:
+                      type: str
+                      required: true
+                      pattern: "\\d+(\\.\\d)?"
+                      convert_types:
+                        - "int"
+                      description: |
+                        Timeout after which port should be errdisabled or
+                        should start dropping on congested priorities.
+                        This should be decimal with up to 1 decimal point
+                        Example : 0.5 or 60
+                    polling_interval:
+                      type: str
+                      required: true
+                      pattern: "auto|\\d+(\\.\\d)?"
+                      convert_types:
+                        - "int"
+                      description: |
+                        Time interval at which the watchdog should poll the queues.
+                        This should be decimal with up to 1 decimal point or set
+                        to 'auto' based on recovery_time and timeout values.
+                        Example : 0.5 or 60
+                    recovery_time:
+                      type: str
+                      required: true
+                      pattern: "\\d+(\\.\\d)?"
+                      convert_types:
+                        - "int"
```

Comment:
```suggestion
                      convert_types:
                        - int
                        - float
```

Reply:
I was not aware that the conversation of float was also supported! This is great. Adding these to the schemas.